Makefile.am

```diff
@@ -63,6 +64,7 @@ libeos_updater_util_libeos_updater_util_@EUU_API_VERSION@_la_CFLAGS = \
 	$(NULL)
 libeos_updater_util_libeos_updater_util_@EUU_API_VERSION@_la_CPPFLAGS = \
 	$(AM_CPPFLAGS) \
+	-DG_LOG_DOMAIN=\""LibEosUpdaterUtil"\" \
```

Comment:
Nitpick: I’d prefer this to be `libeos-updater-util` to match the library filename.